simpledotorg/simple-server
# Use locale to translate 'facility group' in UI

**Story card:** [ch3223](https://app.clubhouse.io/simpledotorg/story/3223/update-new-facility-group-edit-facility-group-view)

## Because

Users don't know what a "facility group" is, so we should show them what they represent (district, zone, upazila, etc.)

## This addresses

This localizes a "facility group" and updates all relevant views to reflect the change.

*Note*: I excluded the CSV facility upload from this because it's tricky to localize headers

## Test instructions

1. Go into `.env.development` and set `DEFAULT_COUNTRY=BD`
2. Go through the Facilities and Organizations admin views on the dashboard
3. Ensure that "facility group" shows up as "upazila" in all relevant places
